Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -328,6 +328,24 @@ const ScrollView = React.createClass({
      * @platform android
      */
     scrollPerfTag: PropTypes.string,
+
+     /**
+     * Used to override default value of overScroll mode.
+     *
+     * Possible values:
+     *
+     *  - `'always'` - Always allow a user to over-scroll this view.
+     *  - `'always-if-content-scrolls'` - Default value, allow a user to over-scroll
```

Comment:
@ericvicenti To give you some background, the names `always`, `always-if-content-scrolls`, and `never` come from the names of the Java constants `OVER_SCROLL_ALWAYS`, `OVER_SCROLL_IF_CONTENT_SCROLLS`, and `OVER_SCROLL_NEVER`.

Would you still prefer if I renamed `always-if-content-scrolls` to `auto`?